ocs_ci/ocs/acm/acm.py

```diff
@@ -41,19 +44,35 @@ def import_cluster_ui(self, cluster_name, kubeconfig_location):
 
         """
         self.navigate_clusters_page()
-        self.do_click(self.page_nav["Import_cluster"])
+        time.sleep(40)
+        self.driver.find_element_by_id("importCluster").click()
+        log.info("Clicked on Import cluster")
+        time.sleep(60)
         self.do_send_keys(
             self.page_nav["Import_cluster_enter_name"], text=f"{cluster_name}"
         )
+        log.info("Keys sent")
+        log.info("sent input to the text box")
         self.do_click(self.page_nav["Import_mode"])
         self.do_click(self.page_nav["choose_kubeconfig"])
         log.info(f"Coping Kubeconfig {kubeconfig_location}")
         kubeconfig_to_import = copy_kubeconfig(kubeconfig_location)
         for line in kubeconfig_to_import:
-            self.do_send_keys(self.page_nav["Kubeconfig_text"], text=f"{line}")
-            time.sleep(2)
+            if len(line) > 100:
+                for c in line:
+                    self.do_send_keys(self.page_nav["Kubeconfig_text"], text=f"{c}")
+            else:
+                self.do_send_keys(self.page_nav["Kubeconfig_text"], text=f"{line}")
+            time.sleep(10)
+        # With ACM2.6 there will be 1 more page
+        # 1. Automation
+        # So we have to click 'Next' button
+        acm_version_str = ".".join(get_running_acm_version().split(".")[:2])
+        if compare_versions(f"{acm_version_str} >= 2.6"):
+            for i in range(2):
```

Comment:
Trying to understand why do we need this line?

Reply:
If you check the above comment , it tells that any ACM less than 2.6 doesn't have this extra page, hence if its >=2.6 we need this extra clicks